upi/azure/06_workers.json

```diff
@@ -44,6 +44,17 @@
       "metadata" : {
         "description" : "The size of the each Node Virtual Machine"
       }
+    },
+    "hyperVGen": {
+      "type": "string",
+      "metadata": {
+        "description": "VM generation image to use"
+      },
+      "defaultValue": "V1",
```

Comment:
the defaultValue is V1 or V2 ?  Now IPI default value is V2.

Reply:
IPI default to V2 if the VM supports it. Unfortunately, we cannot check which HyperVGen the VM supports in the ARM templates, so I'm defaulting to "V1" to keep it consistent with the previous behaviour.

Reply:
Hi @r4f4, V2 is the only one supported for ARM64. Which AMD64 machine types do not support V2?

Is it worth giving those machine types "more weight" than defaulting to V2 and supporting both architectures by default? 

I don't know how many they are, or their usage frequency, and it could be better to just document `if you're deploying on arm64, let's use V2` than `the following *list* of machine types needs V1`.

That's just a bit odd that we need to treat arm64 as "special" again and wanted you to reconsider the choice from this perspective as well.

Reply:
The default is now "V2".